experimental/cli/__main__.py

```diff
@@ -0,0 +1,46 @@
+import logging
```

Comment:
rename to main.py

also, would you be able to add a docstring showing example usage at the top of this file? 

Would be nice to have a readme for this folder as well showing how users can use this

Reply:
Sure, I can add doctorings and README. Regarding the renaming, I use the name `__main__.py` so that I can run it directly with `python -m experimental.cli`, while `main.py` can't. Do you think I can keep using this name?

Reply:
updated, @jerryjliu could you review?